tools/applicationinsights-web-snippet/createAiSnippet.js

```diff
@@ -0,0 +1,39 @@
+// createFile.js
+const fs = require('fs');
+const path = require('path');
+
+// Define the path to the snippet.js file
+const filePath = path.join(__dirname, 'src/snippet.ts'); // Adjust the path if necessary
+const aiFilePath = path.join(__dirname, 'src/aiSnippet.ts');
+const oneDSFilePath = path.join(__dirname, 'src/oneDSSnippet.ts');
+// Read the snippet.js file
+fs.readFile(filePath, 'utf8', (err, data) => {
+    if (err) {
+        console.error('Error reading file:', err);
+        process.exit(1);
+    }
+
+    let remove = data.replace(/let isOneDS = false;/g, '');
```

Comment:
Lets add an additional check after this to make sure that remove != data -- so that if for some reason the regex misses (because of content changes) the build breaks.

it might also be useful to make the regex a little more permissive
`/let\s+isOneDS\s+=\s+\w+;/i